index.html

```diff
@@ -5,9 +5,171 @@
+          </div>
+        </div>
+        <div class="partners-container">
+          <div class="partner-cards">
+            <img class="cards-images" src="images/sega.jpg" alt="">
+          </div>
+        </div>
+        <div class="partners-container">
+          <div class="partner-cards">
+            <img class="cards-images" src="images/bethesda-2.png" alt="">
+          </div>
+        </div>
+        <div class="partners-container">
+          <div class="partner-cards">
+            <img class="cards-images" src="images/bungie.jpg" alt="">
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="contact" id="contact">
+      <div class="container-contact">
+        <div class="row">
+          <h2 class="contact-heading">Stay in touch with us for more News and Special Offers</h2>
+          <p>GAMESCOM sends occasional emails to our community containing event updates,special offers, 
+            and ways you can get involved. If you want to stay connected, subscribe using the form below!
+          </p>
+          <form action="https://formspree.io/f/mdoyvbwr" id="contact-form" class="contact-form">
+            <div class="form-row">
+              <label for="full-name"></label>
+              <input name="full-name" class="form-field" maxlength="30" placeholder="Full name" id="full-name" type="text">
+            </div>
+            <div class="form-row">
+              <label for="email"></label>
+              <input class="form-field" id="email" name="email" type="email" placeholder="Email">
+            </div>
+            <div class="form-row">
+              <button class="button-form" title="submit-btn" type="submit">Subscribe</button>
+            </div>
+          </form>
+        </div>
+      </div>
+    </section>
+    <footer class="footer">
+      <div class="footer-container">
+        <div class="row-footer">
+          <div class="conection">
+            <div class="conection-text">
+              <h3>GAMESCOM&nbsp;<span class="event-text">Event</span></h3>
+              <p>
+                <strong>Creativity unleashed. Act V</strong>
+                <br>
+                17 - 19 November · Brno, Czech Republic 
+              </p>
+              <span class="copy">
+                © 2016-2022
+                GAMESCOM
+              </span>
+            </div>
+            <div class="conection-links">
+              <h3>Conect</h3>
+              <ul class="social-links">
+                <li class="social-links-item"><a href=""><i class="fab fa-facebook hover black fa-2x"></i></a></li>
+                <li class="social-links-item"><a href=""><i class="fab fa-twitter hover black fa-2x"></i></a></li>
+                <li class="social-links-item"><a href=""><i class="fab black hover fa-instagram-square fa-2x"></i></a></li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </div>
+    </footer>
```

Comment:
- [ ] Please follow the same pater design on the footer. 

![Capture](https://user-images.githubusercontent.com/64173189/142462233-3427d148-52de-400a-a0d1-d1935521a225.JPG)


Original design

![Capture](https://user-images.githubusercontent.com/64173189/142462164-23239823-1d4d-429f-936e-013d692a141a.JPG)